### PR TITLE
Added basic TCP-Logger support for multi node tests

### PR DESCRIPTION
**Pull request by Silv3rcircl3**
This PR is a another attempt to implement #1475, currently it's only a starting point for further steps and not ready.

### CODE REVIEW

File: src/core/Akka.MultiNodeTestRunner/Program.cs [hunk 1]
```diff
@@ -71,6 +74,9 @@ static void Main(string[] args)
             TestRunSystem = ActorSystem.Create("TestRunnerLogging");
             SinkCoordinator = TestRunSystem.ActorOf(Props.Create<SinkCoordinator>(), "sinkCoordinator");
 
+            var tcpLogger = TestRunSystem.ActorOf<TcpLoggingServer>();
+            TestRunSystem.Tcp().Tell(new Tcp.Bind(tcpLogger, new IPEndPoint(IPAddress.Any, 5678)));
```
Comment (Horusiath) [inline src/core/Akka.MultiNodeTestRunner/Program.cs]: Does it need to be a hardcoded port? Maybe a config value or program parameter would be a better idea?

File: src/core/Akka.MultiNodeTestRunner/Program.cs [hunk 2]
```diff
@@ -225,5 +231,29 @@ static void PublishToAllSinks(string message)
...
+    class TcpLoggingServer : ReceiveActor
+    {
+        public TcpLoggingServer()
+        {
+            var currentDirectory = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
+            var logFilePath = Path.Combine(currentDirectory, "TcpLog.txt");
+
+            Receive<Tcp.Connected>(connected =>
+            {
+                File.AppendAllText(logFilePath, $"Node connected on {Sender}{Environment.NewLine}");
```
Comment (Horusiath) [inline src/core/Akka.MultiNodeTestRunner/Program.cs]: We're still bound to C# 5 syntax. Interpolated strings are not allowed.

File: src/core/Akka.NodeTestRunner/Program.cs [hunk 3]
```diff
@@ -58,6 +69,9 @@ static void Main(string[] args)
                                 specFail.FailureStackTraces.Add(innerEx.StackTrace);
                             }
                             Console.WriteLine(specFail);
+
+                            //make sure message is send over the wire
+                            Task.Delay(TimeSpan.FromSeconds(10)).Wait();
```
Comment (Horusiath) [inline src/core/Akka.NodeTestRunner/Program.cs]: This is equivalent of `Thread.Sleep(10000)` ;)

File: src/core/Akka.MultiNodeTestRunner/Program.cs (hunk 1 above)
Comment (Aaronontheweb) [inline src/core/Akka.MultiNodeTestRunner/Program.cs]: I wrote a version that does this - let me see if I can dig it up

File: src/core/Akka.MultiNodeTestRunner/Program.cs (hunk 2 above)
Comment (Aaronontheweb) [inline src/core/Akka.MultiNodeTestRunner/Program.cs]: @Horusiath we decided to change that at the last meeting, did we not?

File: src/core/Akka.NodeTestRunner/Program.cs (hunk 3 above)
Comment (Aaronontheweb) [inline src/core/Akka.NodeTestRunner/Program.cs]: I second this change :+1:

**Comment by Aaronontheweb** [comment]
Closing this in favor of #1604

**Pull request closed without merging**